exams/management/commands/populate_edx_exam_coupons.py

```diff
@@ -32,16 +33,22 @@ def handle(self, *args, **kwargs):  # pylint: disable=unused-argument,too-many-l
 
         csvfile = kwargs.get('csvfile')
         reader = csv.DictReader(csvfile.read().splitlines())
-        edx_course_key = next(reader)['Course ID']
+        catalog_query = next(reader)['Catalog Query']
+        course_number = re.search(r"key:\(([A-Za-z0-9.]+)", catalog_query)
```

Comment:
This returns a `Match` object which causes an error below in the `Course.objects.filter` line